src/main.cpp

```diff
@@ -425,8 +425,8 @@ int main(int argc, char* argv[])
         }
 
         // Repeat for other text, but we know it will fit at given size
-        QString sourceCopyrightText = QChar(169) % QString(" Mudlet makers 2008-") % QString(__DATE__).mid(7);
-        QFont font("DejaVu Serif", 16, QFont::Bold | QFont::Serif | QFont::PreferMatch | QFont::PreferAntialias);
+        QString sourceCopyrightText = QStringLiteral("©️ Mudlet makers 2008-2018");
```

Comment:
As per https://github.com/Mudlet/Mudlet/pull/1641/files#r187763701 and the other comment with it.